web/src/app/help/help.component.ts

```diff
@@ -1,12 +1,14 @@
 import {Component, OnInit} from '@angular/core';
+// import {MatButtonModule} from '@angular/material/button';
 
 @Component({
   selector: 'app-help',
   templateUrl: './help.component.html',
-  styleUrls: ['./help.component.scss']
+  styleUrls: ['./help.component.scss'],
+  preserveWhitespaces: true
 })
 export class HelpComponent implements OnInit {
   constructor() {}
 
   ngOnInit() {}
-}
+}
```

Comment:
nit: prefer a newline at the end of the file

Reply:
Any reason for that?

Reply:
https://stackoverflow.com/a/729795 

Mostly historical reasons. For us this is just a stylistic issue. No big deal. I sometimes miss this as well.